### DISCUSSION TITLE
求助   替换 Hosts 中的 Cloudflare CDN IP是替换哪一个   不懂这些~

### DISCUSSION THREAD

**Discussion by Lemon10668** [XIU2/CloudflareSpeedTest#395] [提问]
![image](https://github.com/XIU2/CloudflareSpeedTest/assets/127711727/e46c347f-0b09-4377-a359-2f4da51c78f5)

**Comment by XIU2** [comment]
你要先添加一行内容啊，比如：

```
1.1.1.1 域名
```

然后运行脚本提示时写 `1.1.1.1`，这样脚本运行时，就知道替换 Hosts 里面的所有 `1.1.1.1` 为新 IP 了。

> 你这个 discussions 的邮件通知莫名其妙被归到垃圾邮件了。。。现在才看到

**Comment by Lemon10668** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by Lemon10668** [comment]
谢谢大佬耐心解答，成功解决了！！！







&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "XIU2/CloudflareSpeedTest"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年7月1日(星期六) 晚上8:05
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [XIU2/CloudflareSpeedTest] 求助   替换 Hosts 中的 Cloudflare CDN IP是替换哪一个   不懂这些~ (Discussion #395)





 
你要先添加一行内容啊，比如：
 1.1.1.1 域名  
然后运行脚本提示时写 1.1.1.1，这样脚本运行时，就知道替换 Hosts 里面的所有 1.1.1.1 为新 IP 了。
  
你这个 discussions 的邮件通知莫名其妙被归到垃圾邮件了。。。现在才看到
